content/docs/intro/cloud-providers/aiven/_index.md

```diff
@@ -76,13 +76,22 @@ import (
   aiven "github.com/pulumi/pulumi-aiven/sdk/v2/go/aiven"
 )
 
-service, _ := aiven.NewService(ctx, "test", &aiven.ServiceArgs{
-  Project:     pulumi.String("my-project"),
-  CloudName:   pulumi.String("google-europe-west1"),
-  Plan:        pulumi.String("startup-4"),
-  ServiceName: pulumi.String("my-service"),
-  ServiceType: pulumi.String("grafana"),
-})
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+    service, err := aiven.NewService(ctx, "test", &aiven.ServiceArgs{
+      Project:     pulumi.String("my-project"),
+      CloudName:   pulumi.String("google-europe-west1"),
+      Plan:        pulumi.String("startup-4"),
+      ServiceName: pulumi.String("my-service"),
+      ServiceType: pulumi.String("grafana"),
+    })
+    if err != nil {
+      return err
+    }
```

Comment:
Indentation seems off?  Maybe tabs/spaces issue?

Reply:
Quite a few different variants of these formatting issues throughout the rest of the PR it seems?